File: Semestru 3/MAP/lab/lab10/lab10/lab10/domain/Entity.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace lab10.domain
{
	internal class Entity<ID>
	{
		private ID Id;
		public ID getId() => this.Id;
		public void setId (ID id) => this.Id = id;
	}
}

File: Semestru 4/MPP/lab/java/lab3/Model/src/main/java/org/example/domain/Entity.java
package org.example.domain;

import java.io.Serializable;

public class Entity<ID> implements Serializable {
    private ID Id;
    public Entity() {};

    public ID GetId() {
        return this.Id;
    }

    public Entity<ID> SetId(ID Id) {
        this.Id = Id;
        return this;
    }
}
